File: Day1/PYTHON/FizzBuzz.py
 #    @author: <GitHub Username>
 #    @date: <date>

i=int(input("Enter The No. Of Terms You Want"))
for no in range(1,i+1):
    if no%5==0 and no%3==0:
     print("FizzBuzz",end=' ')
    elif no%5==0:
     print("Buzz",end=' ')
    elif no%3==0:
     print("Fizz",end=' ')
    else:
     print(no,end=' ')

File: Day1/JAVA/FizzBuzz.java
import java.util.Scanner;

public class FizzBuzz {
	
	public static void main(String ar[]) 
	{	
		Scanner scn=new Scanner(System.in);
		System.out.println("Enter The Number Of Terms You Want In The Output");
		int i=scn.nextInt();
		for(int no=1;no<i;no++)
		{
			if(no%15==0)
				System.out.print("FizzBuzz ");
			else if(no%5==0)
				System.out.print("Buzz ");
			else if(no%3==0)
				System.out.print("Fizz ");
			else 
				System.out.print(no+" ");
		}
	}	

}
